mvpa2/tests/test_gnb.py

```diff
@@ -55,6 +56,38 @@ def test_gnb(self):
                             v = np.exp(v)
                         d1 = np.sum(v, axis=1) - 1.0
                         self.assertTrue(np.max(np.abs(d1)) < 1e-5)
+                    if cv:
+                        assert 'has_sensitivity' in gnb_.__tags__
+                        gnb_.get_sensitivity_analyzer()
+                    if not cv:
+                        with self.assertRaises(NotImplementedError):
+                            gnb_.get_sensitivity_analyzer()
+
+
+def test_gnb_sensitivities():
+    gnb = GNB(common_variance=True)
+    ds = normal_feature_dataset(perlabel=4,
+                                nlabels=3,
+                                nfeatures=5,
+                                nchunks=4,
+                                snr=10
```

Comment:
Please add this so there is signal added to those 3 features
```python
nonbogus_features=[0, 1, 2]
```
this would add signal for first label into feature index 0, 2nd label into feature index 1 and so on, so later on you could actually see the effect on sensitivies.